### DISCUSSION TITLE
StreamFX not showing in Flatpak build of OBS 28 (Ubuntu 22.04)

### DISCUSSION THREAD

**Discussion by ItsRogueRen** [Xaymar/obs-stream-effects#906] [General]
Wanna start by saying I love this plugin, but I can't seem to get it working in the flatpak build of OBS after the 28.0 update. I've seen it works in the deb build so I'm not sure what's different. I'm using the latest testing build (as of writing `StreamFX 0.12.0 Alpha 106`) which is listed to support OBS 28.

I have extracked the .7z folder `streamfx-ubuntu-22-0.12.0a106-gb3957959.7z` to the location listed in the installation instructions for the flatpak in `~/.var/app/com.obsproject.Studio/config/obs-studio/` But I get no filters in OBS. I've also checked Flatseal and my OBS has all file permissions. I also tried the other .7z archive `streamfx-ubuntu-20-0.12.0a106-gb3957959.7z ` just to be sure with no change.

**Comment by Xaymar** [comment]
StreamFX does not support Flatpak.

**Reply by ItsRogueRen** [reply]
Sorry, I'm confused. The install instructions tell you how to install on flatpak, when was support dropped and will it be added back?

**Reply by Xaymar** [reply]
As per System Requirements, the only platforms officially supported are Ubuntu 20.04 and Ubuntu 22.04. There have never been any public versions that support Flatpak, though there is a separately maintained version of StreamFX for flatpak. The wiki is community driven and the instructions were community contributed.

**Reply by ItsRogueRen** [reply]
I see now, thank you for the explanation!

**Reply by ItsRogueRen** [reply]
Where can I find this separately maintained version of FtreamFX for flatpak? I cannot find any mention of it anywhere on github

**Reply by Xaymar** [reply]
https://github.com/flathub/flathub/pull/3159
